README.md

```diff
@@ -10,18 +9,17 @@ The implementation is based on other projects:
   * https://github.com/cupertinomiranda/esphome
   * https://github.com/jakosek/esphome/tree/dmx_no_uart
 
-It differs in that it uses the internal UART component but generates the
-break signal by detaching and re-attaching the GPIO pin (on ESP32).
+It differs in that it uses the internal UART component but generates the break signal by detaching and re-attaching the GPIO pin.
 
```

Comment:
the ESP32 statement in brackets is on purpose. The signal is generated differently on the ESP8266. I suggest the removal of this sentence as it is not really necessary.